release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3158,6 +3158,47 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+[id="ocp-4-14-8"]
+=== RHSA-2024:0050 - {product-title} 4.14.8 bug fix and security update
+
+Issued: 2024-01-09
+
+{product-title} release 4.14.8, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:0050[RHBA-2024:0049] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:0052[RHBA-2024:0052] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.8 --pullspecs
+----
+
+[id="ocp-4-14-8-features"]
+==== Features
+
+The following features are included in this z-stream release:
+
+* With this release, Telemetry data is collected from clusters where pod security admission violations are occurring.
+The data being collected is whether the offending namespace is a Kubernetes system namespace, an {product-title} system namespace, or a custom namespace. This data collection will help Red Hat evaluate customer cluster readiness for pod security admission global restricted enforcement in the future. For more information about pod security admission, see xref:../authentication/understanding-and-managing-pod-security-admission.adoc#understanding-and-managing-pod-security-admission[Understanding and managing pod security admission]. (link:https://issues.redhat.com/browse/OCPBUGS-25384[*OCPBUGS-25384*])
+
+* Previously, customers could not leverage OpenShift's Azure Identity capability for short-lived authentication tokens with layered products. With this release, OLM-managed operators have increased security by enabling this support. (link:https://issues.redhat.com/browse/OCPBUGS-25275[*OCPBUGS-25275*])
+
+[id="ocp-4-14-8-bug-fixes"]
+==== Bug fixes
+
+* Previously, installation would fail when installing {product-title} with the `bootMode` set to `UEFISecureBoot` on a node where Secure Boot is disabled. With this release, subsequent attempts to install {product-title} with Secure Boot enabled will proceed normally.(link:https://issues.redhat.com/browse/OCPBUGS-19884[*OCPBUGS-19884*])
+
+* Previously, the installer would fail to destroy a cluster when using regional PDs in google cloud.
```

Comment:
I'm not sure if this should be "Google Cloud Platform", but at the very least should be capitalized as Google Cloud.

```suggestion
* Previously, the installer would fail to destroy a cluster when using regional PDs in Google Cloud.
```